settings/src/lib.rs

```diff
@@ -637,10 +639,12 @@ impl FileWrite for RitaSettingsStruct {
     fn write(&self, file_name: &str) -> Result<(), Error> {
         let ser = toml::Value::try_from(self.clone())?;
         let ser = toml::to_string(&ser)?;
-        let mut file = File::create(file_name)?;
-        file.write_all(ser.as_bytes())?;
-        file.flush().unwrap();
-
+        {
+            let mut file = File::create(file_name)?;
```

Comment:
what do the brackets achieve here?

Reply:
the brackets make sure we close the file handle because it goes out of scope, before we do the sync command